pkg/network/protocols/http2/telemetry.go

```diff
@@ -80,7 +80,7 @@ func newHTTP2KernelTelemetry() *kernelTelemetry {
 		http2responses:                newTLSAwareCounter(metricGroup, "responses"),
 		endOfStream:                   newTLSAwareCounter(metricGroup, "eos"),
 		endOfStreamRST:                newTLSAwareCounter(metricGroup, "rst"),
-		pathExceedsFrame:              newTLSAwareCounter(metricGroup, "path_exceeds_frame"),
+		literalValueExceedsFrame:      newTLSAwareCounter(metricGroup, "literal_value_exceeds_frame"),
```

Comment:
add this to the dashboard

Reply:
added it to the to do in the dashboard. when the metric will be added we can do so